services/workflows-service/src/app.module.ts

```diff
@@ -1,7 +1,7 @@
 import { MiddlewareConsumer, Module } from '@nestjs/common';
 import { APP_GUARD, APP_INTERCEPTOR } from '@nestjs/core';
 import { UserModule } from './user/user.module';
-import { WorkflowModule } from './workflow/workflow.module';
+import { WorkflowControllerModule } from './workflow/workflow-controller.module';
```

Comment:
export controller from the original module, there is no reason for additional module just for the controllers